engine/inc/com/centreon/engine/configuration/state.hh

```diff
@@ -461,6 +467,7 @@ class state {
   void use_timezone(std::string const& value);
   bool use_true_regexp_matching() const noexcept;
   void use_true_regexp_matching(bool value);
+  sched_info_config& sched_info_config() { return _scheduling_info; }
```

Comment:
sched_info_config is private

Reply:
And ??
I don't understand, it works. And why I would set it public if not needed outside?